simpleocr/files.py

```diff
@@ -10,21 +10,8 @@
 GROUND_EXTENSIONS_DEFAULT = GROUND_EXTENSIONS[0]
 
 
-def open_img(path_or_name):
```

Comment:
Let's try and separate the fuzzy file finding mechanism from the class - that way, we can keep the class code minimal